doc/development/deprecations.rst

```diff
@@ -90,6 +90,11 @@ for details on how to port your legacy code to the new system. The following fun
 Completed deprecation cycles
 ----------------------------
 
+* The input argument ``control_wires`` of ``MultiControlledX`` has been marked as deprecated long ago and now removed.
```

Comment:
Just a suggestion to keep the phrasing more consistent with the rest of the section.

```suggestion
* The input argument ``control_wires`` of ``MultiControlledX`` has been removed.
```